src/main/scala/beam/utils/scenario/DataExchange.scala

```diff
@@ -0,0 +1,42 @@
+package beam.utils.scenario
+
+sealed trait InputType {
+
+  def toFileExt: String = this match {
+    case InputType.Parquet => "parquet"
+    case InputType.CSV     => "csv"
+  }
+}
+
+object InputType {
+  case object Parquet extends InputType
+  case object CSV extends InputType
+
+  def apply(`type`: String): InputType = {
```

Comment:
I'm not a fan of using a keyword for a parameter. Just because you _can_ doesn't mean you should.